crates/via-serve-static/src/resolve.rs

```diff
@@ -6,7 +6,12 @@ use std::{
 };
 use tokio::task;
 
-const CHUNK_SIZE: usize = 1024 * 1024 * 10;
+/// The maximum file size that will be eagerly read into memory. As a performance
+/// optimization, small files will be read into memory immediately to avoid the
+/// overhead of calling `tokio::task::spawn_blocking` multiple times.
+///
+/// The current value is set to `10MB`.
+const EAGER_READ_THRESHOLD: u64 = 1024 * 1024 * 10;
```

Comment:
Since we're not actually streaming `10MB` chunks I decided to change the variable name to something more descriptive. I also thought it was a good idea to change the type def to `u64` to match the return value of `Metadata::len` to prevent us from having to do a type cast for each request.

Reply:
`eager_read_threshold` is a configurable option now. I also changed the default value to `1MB`.